File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 11/IPayAsMyself.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_11
{
	internal class IPayAsMyself : StoryChoice
	{
		internal IPayAsMyself() : base(StoryChoice.GetLocalizedString("ch11IPayAsMyself"), 11062)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch11PayedForHotelRoom = HotelRoomPayment.KadenMoney;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_11/IPayAsMyself.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_11;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.HotelRoomPayment;

/**
 * Created by christopherraleigh on 2016-05-22.
 */
public class IPayAsMyself extends StoryChoice {
    public IPayAsMyself() {
        super(R.string.ch11IPayAsMyself, 11062);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh11PayedForHotelRoom(HotelRoomPayment.KadenMoney);
        super.select();
    }
}
